android/src/main/java/com/mrousavy/camera/CameraView.kt

```diff
@@ -235,6 +235,8 @@ class CameraView(context: Context, private val frameProcessorThread: ExecutorSer
       override fun onHostResume() {
         hostLifecycleState = Lifecycle.State.RESUMED
         updateLifecycleState()
+        // workaround for https://issuetracker.google.com/issues/147354615, preview must be bound on resume
+        update(propsThatRequireSessionReconfiguration)
```

Comment:
This may be a messy fix, I know that it solves my problem with preview configuration, but I have not tested sufficiently and am not skilled enough in Android to know that it doesn't introduce other problems.